readthedocs/projects/models.py

```diff
@@ -950,28 +950,33 @@ def active_versions(self):
             versions.filter(active=True, uploaded=True)
         )
 
-    def ordered_active_versions(self, user=None):
+    def ordered_active_versions(self, **kwargs):
```

Comment:
It would be good to have a docstring or comment saying that this `kwargs` argument has to match the filter for the Version, or probably better to just use all the keyword argument that support `.public` directly here.

Probably the second option is better "self-documented".

Reply:
We can't do 2, in .com we override .public to support more kwargs